packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -419,6 +428,11 @@ class MessageList extends Widget<Properties> {
         break;
       case 'isLoading':
         break;
+      case 'showAvatar':
+      case 'showUsername':
+      case 'showMessageTimestamp':
+        this._invalidate();
```

Comment:
[copypaste]
Can be moved above near the `case 'showDayHeaders':`

Reply:
@Strider2342  it's still actual

Reply:
a5e560880a0b06e5dc198898d2ff27fbd277684d